VidyoLog.js

```diff
@@ -439,11 +485,11 @@ function VidyoLog(containerId) {
 			}
 		}
 		var colorRow = " "
-		if (logLine.level == "WARNING") {
+		if (logLine.level == "WARNING" ||logLine.level == "WRN" ) {
 			colorRow = "table-warning ";
-		} else if (logLine.level == "ERROR") {
+		} else if (logLine.level == "ERROR" || logLine.level == "ERR") {
 			colorRow = "table-danger ";
-		} else if (logLine.level == "SENT") {
+		} else if (logLine.level == "SENT" || logLine.level == "SNT") {
 			colorRow = "table-success ";
 		} else if (logLine.level == "RECEIVED") {
```

Comment:
New strings in code --

  case LMI_LOGLEVEL_FATAL: return "FTL";
  case LMI_LOGLEVEL_ERROR: return "ERR";
  case LMI_LOGLEVEL_WARNING: return "WRN";
  case LMI_LOGLEVEL_INFO: return "INF";
  case LMI_LOGLEVEL_DEBUG: return "DBG";
  case LMI_LOGLEVEL_SENT: return "SNT";
  case LMI_LOGLEVEL_RECEIVED: return "RCV";
  case LMI_LOGLEVEL_ENTER: return "ENT";
  case LMI_LOGLEVEL_LEAVE: return "LEV";

Should we add "RCV" as well?